### DISCUSSION TITLE
[@types/yargs] Node16 import resolution missing types

### DISCUSSION THREAD

**Discussion by brandonryan** [DefinitelyTyped/DefinitelyTyped#61189] [Issues with a @types package]
When moduleResolution is set to node16 in my tsconfig, the imports that are available to me are different.
Specifically `ArgumentsCamelCase` and `terminalWidth` exports are missing from `index.d.mts`. 
I need access to ArgumentsCamelCase because i use it to split up my commands into separate files without breaking types.

[a comment by an automated account is not shown]

**Comment by fluffy-heinzelman** [comment]
@brandonryan I had the same problem using `"moduleResolution": "NodeNext"` and were able to import types, including `ArgumentsCamelCase`, using TypeScript-specific `import module = require("module")` syntax.

```ts
import type yargs = require('yargs');

type Foo = yargs.ArgumentsCamelCase<Bar>;
```

TypeScript docs:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require
